os/os-linux.h

```diff
@@ -172,14 +176,27 @@ static inline unsigned long long os_phys_mem(void)
 
 static inline void os_random_seed(unsigned long seed, os_random_state_t *rs)
 {
+#if defined(__GLIBC__) | defined (__UCLIBC__)
```

Comment:
Did you intend to use a single pipe symbol here?

Reply:
Typo. Will fix.